user-apps/demo-pdf/default.nix

```diff
@@ -0,0 +1,28 @@
+# Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  stdenv,
+  pkgs,
+  lib,
+  ...
```

Comment:
You are not using `pkgs` and `lib`, and you don't need `...`.

However, you could add `zathura` here instead of the above